.github/problem-matcher-lint-check-licences.json

```diff
@@ -0,0 +1,16 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "lint-check-licences",
+            "pattern": [
+                {
+                    "regexp": "^(notice|error)(:(file|dir):([^:]+)(:lines? (\\d+)(-(\\d+))?)?)?: (.+)$",
+                    "severity": 1,
```

Comment:
I don't think GitHub recognises notice as a severity, just error and warning.
https://github.com/actions/toolkit/blob/master/docs/problem-matchers.md#single-line-matchers

Reply:
It does! I found it in the [source](https://github.com/actions/runner/blob/aaf02ab34cf874e67c43fc418338db21679e5d52/src/Runner.Worker/Handlers/OutputManager.cs#L218-L221). It causes nice messages like these: https://github.com/DaAwesomeP/ola/actions/runs/4218995033

Reply:
:heart: :facepalm: 
Perhaps you'd care to open a PR to improve their docs?

Reply:
Will do!